documentation/asciidoc/computers/configuration/kernel-command-line-config.adoc

```diff
@@ -6,7 +6,7 @@ The Linux kernel accepts a command line of parameters during boot. On the Raspbe
 sudo nano /boot/cmdline.txt
 ----
 
-Note that we have to use `sudo` to edit anything in the boot partition, and that all parameters in `cmdline.txt` must be on the same line (no carriage returns).
+NOTE: We have to use `sudo` to edit anything in the boot partition, and that all parameters in `cmdline.txt` must be on the same line (no carriage returns).
```

Comment:
Should the "that" be removed?

Reply:
Oh the second "that", yes it should. Fixed in a191a74.